prime-router/metadata/fhir_mapping/hl7/message/ORU_R01.yml

```diff
@@ -8,11 +8,13 @@ resources:
     additionalSegments:
       - SFT
 
-  - resourceName: Provenance
+  - resourceName: Provenance-Source
```

Comment:
So I think this is suppose to be the name of the FHIR resource... did you hit an issue if they all have the same name?

Reply:
So I think we should update it to match the actual resource name just based on the documentation